### ISSUE TITLE
[GUI] Write status screen

### ISSUE THREAD

**Issue by Ximidar** [Ximidar/Flotilla#29]
# Write Status Screen
## write the status screen for the GUI. This status screen should include the following:
 - Temperature Graphs
 - Motor Control
 - Temperature Control
 - Print Status
 - Pause / Resume / Cancel

That will do for a start.

For extra you can find a cool way to display `Flotilla` on the main screen while nothing is happening

**Comment by Ximidar** [comment]
The Comm Screen is giving me issues. I think when things slow down the comm window won't be as laggy, but as for right now it does not render very well. 

I need to find a better solution for managing and displaying the lines that are received. If I don't curb this number it just keeps growing and getting worse. However my current method of trying to limit the size of the array for lines causes lag as well. Add that on top of the fact that every time I add a new line I ask the scroll bar to scroll to the bottom. All of this leads to poor performance. I should figure out a way to improve this. 

Basically this is a problem with my familiarity with JS and vue.

I will try other methods of rendering besides using the vue-for loop.

**Comment by Ximidar** [comment]
I should attempt to write the pause / resume / cancel buttons. I've really wanted them for a while
